staging/src/k8s.io/kubectl/pkg/cmd/expose/expose.go

```diff
@@ -390,15 +388,15 @@ func (o *ExposeServiceOptions) createService() (*corev1.Service, error) {
 		}
 	}
 
-	name := o.Name
-	if len(name) == 0 {
-		name = o.DefaultName
-		if len(name) == 0 {
+	serviceName := o.Name
+	if len(serviceName) == 0 {
+		serviceName = o.DefaultName
+		if len(serviceName) == 0 {
```

Comment:
This is to avoid the misunderstanding of duplicate names. In the loop, name is also used. To avoid misunderstanding, I renamed it